templates/_macros/hero.jinja

```diff
@@ -72,10 +72,7 @@
   {% endif %}
 
   {%- macro _hero_title_block() -%}
-    {%- if layout == '25-75' and has_signpost_image -%}
-      {#- On 25-75 signpost, header top should align with top of the signpost to its left. -#}
-      {% set title_class = "u-no-padding--top" %}
-    {%- elif has_full_width_image -%}
+    {%- if has_full_width_image -%}
```

Comment:
Not sure how this change is related to quote?

Reply:
This is a drive-by to fix something we missed in design review on hero, as requested by Lyubo [here](https://chat.canonical.com/canonical/pl/1bda8imr5bymbde47unn8hjmoe)